README.md

```diff
@@ -27,12 +30,34 @@ The app is completely functional across all devices - from iPhone to full-sized
 
 ![alt-text](http://www.tylermcginnis.com/images/cfh3.png "Desktop with Cards")
 
-Our Tech Stack
+Installation
+----------
+* Ensure you have `git`, `node`, `npm` and `mongodb` correctly installed on your computer.
+* Download or clone this repo to your local computer.  Run `npm install` to install all prerequisite dependecies and frameworks.
+* Create a `.env` file. Update the `MONGOHQ_URL` field with a link to your development database.
+* Run `npm start` to start the app.
```

Comment:
I think new engineers need to start the mongodb server `sudo mongod` before running `npm start` command